examples/gno.land/p/matijamarjanovic/charts/column_chart.gno

```diff
@@ -0,0 +1,113 @@
+package charts
+
+import (
+	"strings"
+
+	"gno.land/p/demo/ufmt"
+)
+
+// GenerateColumnChart creates an ASCII column chart in markdown format
+// values: slice of float values to chart
+// maxColumns: maximum number of columns to display (will normalize if exceeded)
+// title: chart title
+// xAxisTitle: title for the x-axis
+// yAxisTitle: title for the y-axis
+// Returns a markdown string representing the chart
+func GenerateColumnChart(values []float64, maxColumns int, title, xAxisTitle, yAxisTitle string) string {
```

Comment:
The `maxColumns` is defined as `int` type, and it is possible to get values less than 0. need to checking the value or changing the type to `uint`.